src/Cli.Tests/EndToEndTests.cs

```diff
@@ -105,7 +105,7 @@ public void TestInitForCosmosDBPostgreSql()
     public void TestInitializingRestAndGraphQLGlobalSettings()
     {
         string[] args = { "init", "-c", TEST_RUNTIME_CONFIG_FILE, "--database-type", "mssql", "--rest.path", "/rest-api",
-                          "--rest.disabled", "--graphql.path", "/graphql-api" };
+                          "--runtime.base-route", "/base-route", "--rest.disabled", "--graphql.path", "/graphql-api" };
```

Comment:
This argument seems out of place for testing/validation since this test is checking rest/graphql global settings behavior.